test/recipe/admin/pages/specs/error-page.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <UiSpec
+    title="Error Page"
+    :variants="variants"
+    :no-loading-variant="true"
+  />
+</template>
+<script setup>
+import { computed, ref } from "vue";
+import { createError } from "#app";
+import giveMe from "~/utils/giveMe";
+
+const variants = [
+  {
+    name: "Basic",
+    descriptor: giveMe.anErrorPage({
+      safeLinks: [
+        giveMe.aCardLink({ route: "/", title: "Home" }),
+        giveMe.aCardLink({ route: "/specs", title: "Specs" })
+      ],
+      errorResults: [
+        { statusCode: 403, title: "Access Denied", message: "You do not have the permision to view the address or data specified." },
```

Comment:
this should be built using another stubber just like `giveMe.aHeaderItem` in `header.vue`